src/axom/spin/tests/spin_bvh.cpp

```diff
@@ -1306,6 +1306,150 @@ void check_find_points_zip2d()
   axom::setDefaultAllocator(current_allocator);
 }
 
+//---------------------------------------------------------------------------
+
+/*!
+ * \brief Makes an array of 2D points for testing the BVH.
+ *
+ * \return points The array of points to use for testing.
+ */
+template <typename FloatType>
+axom::Array<axom::primal::Point<FloatType, 2>> make_query_points_2d()
+{
+  int dims[] = {101, 101};
+  const FloatType x0 = -1.5, x1 = 1.5;
+  const FloatType y0 = -1.5, y1 = 1.5;
+  axom::Array<axom::primal::Point<FloatType, 2>> points;
+  points.reserve(dims[0] * dims[1]);
+  for(int j = 0; j < dims[1]; j++)
+  {
+    FloatType tj =
+      static_cast<FloatType>(j) / static_cast<FloatType>(dims[1] - 1);
+    FloatType y = (1. - tj) * y0 + tj * y1;
+    for(int i = 0; i <= dims[0]; i++)
+    {
+      FloatType ti =
+        static_cast<FloatType>(i) / static_cast<FloatType>(dims[0] - 1);
+      FloatType x = (1. - ti) * x0 + ti * x1;
+      points.push_back(axom::primal::Point<FloatType, 2>({x, y}));
+    }
+  }
+  return points;
+}
+
+//---------------------------------------------------------------------------
+
+/*!
+ * \brief Tests inserting a single point into the BVH and querying a set of
+ *        points against it, using a pattern seen in DistributedClosestPoint.
+ *        The test demonstrates current_node that the BVH returns an in range
+ *        value to the checkMinDist lambda.
+ *
+ * \param [in] bvh The BVH object to use.
+ * \param [in] query_pts The array of query points to use against the BVH. 
+ */
+template <typename BVHType, typename PointType>
+void bvh_one_bbox_2d(BVHType& bvh,
+                     axom::Array<PointType>& points,
+                     axom::Array<PointType>& query_pts,
+                     bool none)
```

Comment:
Please document the `none` and `points` parameters.

Based on the code below, it looks like the `none` parameter allows the code to either use the point or not.
Since the function takes an array, maybe it should be renamed to something like `bvh_compute_point_distances_2d` (or something better)?